lib/elements/element.py

```diff
@@ -153,6 +160,10 @@ def has_style(self, style_name):
     def stroke_scale(self):
         svg = self.node.getroottree().getroot()
         doc_width, doc_height = get_doc_size(svg)
+        # this is necessary for clones, since they are disconnected from the DOM
+        # it will result in a slighty wrong result for zig-zag stitches
+        if doc_width == 0:
+            doc_width = 1
```

Comment:
Maybe it's better to just do `return 1` here?  I wish I could think of a better way to handle this.  Maybe we should be storing the viewbox globally or something...

Reply:
Ok, let's return 1 directly.
Let's think about it, when we move to the new Inkscape version?!?

Reply:
Sounds good :)